### PR TITLE
nextcloudPackages: update

### PR DESCRIPTION
**Pull request by dotlambda**
###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/servers/nextcloud/packages/25.json [hunk 1]
```diff
@@ -10,29 +10,29 @@
     ]
   },
   "calendar": {
-    "sha256": "0xhrpadzz73rdjyk4y1xm5hwc6k104rlpp9nmw08pq8phpfs12qa",
-    "url": "https://github.com/nextcloud-releases/calendar/releases/download/v4.3.3/calendar-v4.3.3.tar.gz",
-    "version": "4.3.3",
+    "sha256": "154xw925v8wipl472vpksiccf1jnic615add6v1zzaxwqwclhjrv",
+    "url": "https://github.com/nextcloud-releases/calendar/releases/download/v4.4.0-beta1/calendar-v4.4.0-beta1.tar.gz",
+    "version": "4.4.0-beta.1",
```
Comment (RaitoBezarius) [inline pkgs/servers/nextcloud/packages/25.json:15]: Is there a good reasaon to upgrade to betas?

Comment (SuperSandro2000) [reply pkgs/servers/nextcloud/packages/25.json:15]: the script just updates to the latest available version regardless of it is a beta

Comment (onny) [reply pkgs/servers/nextcloud/packages/25.json:15]: Tracked the issue here https://nixos.wiki/wiki/Nextcloud#Known_issues , upstream https://github.com/helsinki-systems/nc4nix/issues/4 and trying to create a PR for it to fix it :)

Comment (RaitoBezarius) [reply pkgs/servers/nextcloud/packages/25.json:15]: They use GitLab, this is a readonly mirror

**Pull request merged**